tests/e2e/system_test/mon/test_restore_ceph_mon_quorum.py

```diff
@@ -0,0 +1,166 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    system_test,
+    polarion_id,
+    skipif_ocs_version,
+    ignore_leftovers,
+)
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import TimeoutExpiredError, ResourceWrongStatusException
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import (
+    wait_for_storage_pods,
+    get_ceph_tools_pod,
+    get_mon_pods,
+)
+from ocs_ci.helpers.helpers import (
+    induce_mon_quorum_loss,
+    recover_mon_quorum,
+    modify_deployment_replica_count,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@system_test
+@skipif_ocs_version("<4.9")
+@polarion_id("OCS-2720")
+class TestRestoreCephMonQuorum(E2ETest):
+    """
+    The objective of this test case is to verify that
+    mons can be brought to quorum successfully by
+    following the steps mentioned in
+    https://access.redhat.com/solutions/5898541, when
+    mon(s) are out of quorum on a cluster
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.fixture(autouse=True)
+    def base_set_up(self):
+        """
+        Preconditions:
+
+        a) Have a cluster with FIPS and Hugepages enabled
+        b) Create some resources s3 objects, buckets and write data's to it.
+        c) Create RGW kafka notification & see the objects are notified to kafka
+        d) Perform mcg bucket replication (bidirectional) and see the objects are synced.
+        e) Perform noobaa caching
+        f) there are snapshots and clones for few PVCs
+        g) Background IOs (fio pods) running
+        """
+        # ToDo:Validate FIPS, hugepages enabled in cluster
```

Comment:
These should be validated after successful deployment, so I think we can remove the validation from test case here.

Reply:
ack